src/modules/dashboard/components/ColonyHomeActions/ColonyActionsDialog.css

```diff
@@ -0,0 +1,20 @@
+.content {
+  padding-bottom: var(--pad-huge);
+}
+
+.header {
+  position: relative;
+  padding: 0 40px;
+  padding-top: var(--pad-huge);
+  padding-bottom: var(--pad-large);
```

Comment:
When you have values like this _(eg: `padding`, `margin`)_, try to combine them and use the shorthand version.

In this case this will be:
```css
padding: var(--pad-huge) 0 var(--pad-huge);
```